### DISCUSSION TITLE
beets removed?

### DISCUSSION THREAD

**Discussion by krokenn** [RandomNinjaAtk/docker-lidarr-extended#95] [General]
hello, is beeets removed? im trying to get it to work but using latest it is not working.

**Comment by RandomNinjaAtk** [comment]
Its still installed, but no longer used because it was causing to many tagging issues in my experience... Your able to write your own script or use it manually via CLI if you desire.

**Reply by Kolbasz12** [reply]
Any chance you can elaborate on the issues you were having?  Was it a yaml config issue or something else as I thought it uses the same musicbrainz source for its data. Thanks, just hoping to understand.

**Reply by RandomNinjaAtk** [reply]
Tags are frankly a bit messy when using beets. It often would add multiple album artists which lead to undesirable results in apps such as jellyfin.

Picard is my preferred tagger, but sadly there is no way to automate it that I can find.

Edit: Also the intended use-case was to pre-match albums to improve lidarrs chance of correctly identifying and successfully importing the downloaded album, which worked but had the undesirable side affects by beets messy tagging...

**Reply by Kolbasz12** [reply]
That’s most interesting as I have long used beets to manage my library and tags. I liked the offer of lidarr and was exploring it. 

Headphones back in the day never did it for me. 

One negative of lidarr is disc numbering vs continuous track numbers. A plus of beets is the ability to source tags from discogs and beat port. 

I was excited to combine the two, so it’s unfortunate to see you had issues and disabled the integration. Do you plan to keep it installed for all future releases?

**Reply by RandomNinjaAtk** [reply]
There are no plans to remove it at this time.

**Reply by Kolbasz12** [reply]
Glad to hear it. I’ll just plan to first pass with lidarr and then call beets with  a sh script. Thanks
